### ISSUE TITLE
QL won't find my custom plugin when I import my auxiliary files (GNU/Linux)

### ISSUE THREAD

**Issue by rakamanja** [quodlibet/quodlibet#3311]
Hello, I'm trying to make a custom plugin on Debian GNU/Linux. The plugin is based on the events/viewlyrics.py plugin. 

If I simply change the plugin's name, id and some of the displayed messages, it gets recognized just fine. If I try to import the auxiliary files to fetch things from the network, then the plugin isn't recognized anymore. 

I tried putting the auxiliary files under ~/.config/quodlibet/plugins/files.py and ~/.config/quodlibet/plugins/package/. It didn't work. In the later case there also was a __init__.py file in the package directory.
I tried putting the auxiliary files under ~/.local/lib/python3.7/sitepackeges and it didn't work either.

Other external imports like `import os` work just fine.  Launching quodlibet from the command line gives no errors, just some warnings about setting XFree86 keys or something.

I can import and use the auxiliary files from ipython.

Note: At first I couldn't import the plugin at all, I had to create a __init__.py file under ~/.config/quodlibet/plugins for it to work. This is nor mentioned in the docs.

**Comment by declension** [comment]
TBH that route of plugin development is a bit unsupported now. 

You'll have _much_ more luck if you check out a version or (`HEAD` of `master` perhaps) from Github and add your plugin to there, having set things up as per [the development guide](https://quodlibet.readthedocs.io/en/latest/development/index.html)

**Comment by rakamanja** [comment]
>that route of plugin development
I'm not sure I understand what this means. I tried downloading master and putting the plugin under ext/events. Is that what you meant? In which case, the plugin got ignored. I assume I have to register it somewhere.

I did try copy pasting viewlyrics.py (from master) and changing the plugin id and name. It still didn't get recognized. So I probably need to register it somewhere, but where?

Putting it under ~/.local/.config/quodlibet/plugins still gives the same results.

Thanks for helping. That's a very spooky profile picture you have btw.

**Comment by declension** [comment]
> I'm not sure I understand what this means. I tried downloading master and putting the plugin under ext/events. Is that what you meant? 

Yes, but remember to _run_ that version as well (e.g. from the command line: `quodlibet/quodlibet.py --debug`)

If `--debug` doesn't reveal anything interesting, then you should click `Show Errors` within the plugin window, which will show you registration errors:
![Screenshot from 2020-03-01 21-23-29](https://user-images.githubusercontent.com/3322808/75634164-0b273e80-5c03-11ea-93d0-b286c232a4e6.png)


If all else fails try [the QL Gitter channel](https://gitter.im/quodlibet/Lobby), but it's not super busy.
